docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -10,9 +10,58 @@ import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
 # Playback a video
 
 To playback your live or on demand video inside your application you'll need to
-use a video player component that supports the HLS protocol. While there are
-many HLS video players out there, we recommend using
-[livepeer.js](https://livepeerjs.org).
+use a video player component that supports the HLS protocol. 
+
+HLS is a standard protocol for streaming video over the internet. Most of the videos 
+you watch on the internet, both live video and on-demand video is delivered over HLS. 
+Livepeer delivers your videos in this standard format.
+
+While there are many HLS video players out there, we recommend using [livepeer.js](https://livepeerjs.org).
+
+## Step 1: Get the `playbackId`
+
+Get the `playbackId` of an existing asset. The `playbackId` can be found in the
+response object of any API call working with assets. If you haven't created an
+asset yet, you can follow the
+[upload a video asset guide](/guides/on-demand/upload-video-asset#api) to do so.
+
+## Step 2: Use the `playbackId` in a player
```

Comment:
We should have instructions on how to create the playback URL from the playbackId as well.